presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -417,6 +417,30 @@ public void testShowCreateForPartitionedTablesWithDataCompression()
                         ")");
     }
 
+    @Test
+    public void testShowCreateForIndexedNonPartitionedTablesWithDataCompression()
+    {
+        sqlServer.execute("CREATE TABLE IndexedTable (\n" +
+                "   key1 BIGINT NOT NULL,\n" +
+                "   key2 BIGINT NOT NULL,\n" +
+                "   key3 BIGINT NOT NULL,\n" +
+                "   key4 BIGINT NOT NULL,\n" +
+                "   key5 BIGINT NOT NULL,\n" +
+                "   CONSTRAINT PK_IndexedTable PRIMARY KEY CLUSTERED (key1),\n" +
```

Comment:
Maybe it worth to create separate test `testShowCreateForIndexedTablesAreNotSupportedYet` just to underline that current test fails not because of data compression once such support is implemented?